apps/backend/prisma/schema.prisma

```diff
@@ -17,4 +17,24 @@ model User {
   password String
   //   posts   Post[]
   //   profile Profile?
+}
+
+model Question {
+  id Int      @id @default(autoincrement())
+  title String
+  description String
+  categories Category[]
+  complexity Complexity
+}
+
+model Category {
+  id Int @id @default(autoincrement())
+  name String
+  questions Question[]
+}
```

Comment:
Can I just check for this part I thought is handled by MongoDB or psql side also handling questions? @igezt

Reply:
This is just for the future when we move over to psql